frontend-react/src/hooks/UseLookupTable.test.ts

```diff
@@ -0,0 +1,63 @@
+import { renderHook } from "@testing-library/react-hooks";
+
+import { lookupTableServer } from "../__mocks__/LookupTableMockServer";
+import { LookupTables, ValueSet } from "../network/api/LookupTableApi";
+
+import useLookupTable, {
+    generateUseLookupTable,
+    GetLatestData,
+    GetLatestVersion,
+    getSenderAutomationData,
+} from "./UseLookupTable";
+
+jest.mock("@okta/okta-react", () => ({
+    useOktaAuth: () => {
+        const authState = {
+            isAuthenticated: true,
+        };
+        return { authState: authState };
+    },
+}));
+
+describe("test all hooks and methods", () => {
+    beforeAll(() => lookupTableServer.listen());
+    afterEach(() => lookupTableServer.resetHandlers());
+    afterAll(() => lookupTableServer.close());
+
+    test("test GetLatestVersion", async () => {
+        let version = await GetLatestVersion(LookupTables.VALUE_SET);
+        expect(version).toEqual(2);
```

Comment:
can we assert a few more things either in this test or subsequent ones? I'd love to see the filter and sort  functionality tested. Maybe a bridge too far, but would also love to see an assertion that axios is being called using the result of `lookupTableApi.getTableList()` and that error states are being handled properly.

Reply:
@doug-s-nava let's add these to the To Do list for #5442